test/WasmSpec/convert-test-suite/index.js

```diff
@@ -114,7 +114,7 @@ function main() {
     const runs = specFiles.map(specFile => {
       const ext = path.extname(specFile);
       const basename = path.basename(specFile, ext);
-      const isXplatExcluded = config["xplat-excludes"].indexOf(path.basename(specFile, ext)) !== -1;
+      const isXplatExcluded = true || config["xplat-excludes"].indexOf(path.basename(specFile, ext)) !== -1;

```

Comment:
well, when we'll want to enable those tests on xplat, we'll have to find this otherwise the script will always output `exclude_xplat`